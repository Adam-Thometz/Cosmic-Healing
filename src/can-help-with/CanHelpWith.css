.CanHelpWith {
  display: flex;
  align-items: center;
}

.CanHelpWith-text-wrapper {
  width: 35%;
  margin: 0 auto;
  text-align: left;
}

.CanHelpWith-moving-text {
  background: -webkit-linear-gradient(45deg, #A3000E, #FBAD7D);
  background-clip: text;
  -webkit-text-fill-color: transparent;
}