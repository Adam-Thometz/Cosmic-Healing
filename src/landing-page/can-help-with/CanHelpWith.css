.CanHelpWith {
  --height: 36px;
  display: flex;
  align-items: center;
  margin: var(--margin) 0;
  animation: slide-up var(--duration) calc(var(--delay) * var(--i)) both;
}

.CanHelpWith-img-wrapper {
  width: 40%;
}

.CanHelpWith-img-wrapper img {
  width: 100%;
}

.CanHelpWith-text-wrapper {
  width: 40%;
  margin: 0 auto;
  text-align: left;
}

.CanHelpWith-slider-wrapper {
  width: 70%;
  height: var(--height);
  position: relative;
}

.CanHelpWith-slider-text {
  color: #E35662;
  font-size: calc((var(--height) / 3) * 2);
  padding: 8px 0;
  margin: 0;
}


.CanHelpWith-slider-wrapper::before {
  content: '';
  z-index: 2;
  background-color: #fff;
  display: block;
  width: 500px;
  height: 100%;
  position: absolute;
  top: 0;
  animation: openclose 5s infinite both;
}

@media (max-width: 600px) {
  .CanHelpWith {
    flex-direction: column;
  }

  .CanHelpWith-text-wrapper {
    width: 80%;
  }

  .CanHelpWith-moving-text {
    display: block;
  }
}

@keyframes openclose {
  from {
    left: -100px;
  }
  20% {
    left: 350px;
  }
  80% {
    left: 350px;
  }
  to {
    left: -100px;
  }
}

@keyframes movetext {
  from {
    margin-bottom: 0;
  }
  25% {
    margin-bottom: calc(var(--height));
  }
  30% {
    margin-bottom: calc(var(--height));
  }
  50% {
    margin-bottom: calc(var(--height) * 2);
  }
  55% {
    margin-bottom: calc(var(--height) * 2);
  }
  65% {
    margin-bottom: calc(var(--height) * 3);
  }
  70% {
    margin-bottom: calc(var(--height) * 3);
  }
  950% {
    margin-bottom: calc(var(--height) * 4);
  }
}