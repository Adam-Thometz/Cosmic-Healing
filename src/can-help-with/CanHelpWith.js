import React from 'react';

import './CanHelpWith.css'

import SmallCaption from '../_components/small-caption/SmallCaption';
import SectionHeader from '../_components/section-header/SectionHeader';

import hand from '../_media/hand.png';

const CanHelpWith = () => {
  const headerText = <>I can help improve <span className='CanHelpWith-moving-text'>emotional intelligence</span></>

  return (
    <section className='CanHelpWith'>
      <aside className='CanHelpWith-img-wrapper'>
        <img src={hand} alt='' />
      </aside>
      <aside className='CanHelpWith-text-wrapper'>
        <SmallCaption text="UNDERSTAND YOURSELF AND YOUR PURPOSE" />
        <SectionHeader text={headerText} />
      </aside>
    </section>
  );
};

export default CanHelpWith;