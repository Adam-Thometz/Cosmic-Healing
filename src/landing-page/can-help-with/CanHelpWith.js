import React from 'react';

import './CanHelpWith.css'

import SmallCaption from '../../_components/small-caption/SmallCaption';
import SectionHeader from '../../_components/section-header/SectionHeader';

import hand from '../../_media/hand.png';

const CanHelpWith = () => {
  const headerText = <>Learn how to <span className='CanHelpWith-moving-text'>improve emotional intelligence</span></>

  return (
    <section className='CanHelpWith'>
      <aside className='CanHelpWith-img-wrapper'>
        <img src={hand} alt='' />
      </aside>
      <aside className='CanHelpWith-text-wrapper'>
        <SmallCaption text="UNDERSTAND YOURSELF AND YOUR PURPOSE" />
        <SectionHeader text={headerText} />
      </aside>
    </section>
  );
};

export default CanHelpWith;