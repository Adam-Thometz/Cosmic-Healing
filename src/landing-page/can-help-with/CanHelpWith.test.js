import React from "react";
import { render } from "@testing-library/react";
import CanHelpWith from './CanHelpWith';

describe('CanHelpWith component', () => {
  it('renders without crashing', () => {
    render(<CanHelpWith />);
  });

  it('matches the snapshot', () => {
    const { asFragment } = render(<CanHelpWith />);
    expect(asFragment()).toMatchSnapshot();
  });
});