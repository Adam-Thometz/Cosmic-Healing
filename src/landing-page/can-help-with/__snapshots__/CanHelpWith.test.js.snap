// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CanHelpWith component matches the snapshot 1`] = `
<DocumentFragment>
  <section
    class="CanHelpWith"
  >
    <aside
      class="CanHelpWith-img-wrapper"
    >
      <img
        alt=""
        src="hand.png"
      />
    </aside>
    <aside
      class="CanHelpWith-text-wrapper"
    >
      <small
        class="SmallCaption"
        style="color: black;"
      >
        UNDERSTAND YOURSELF AND YOUR PURPOSE
      </small>
      <h2
        class="SectionHeader"
      >
        Learn how to 
        <span
          class="CanHelpWith-moving-text"
        >
          improve emotional intelligence
        </span>
      </h2>
    </aside>
  </section>
</DocumentFragment>
`;
